packages/replay/src/replay.ts

```diff
@@ -223,10 +223,9 @@ export class ReplayContainer implements ReplayContainerInterface {
       if (this._stopRecording) {
         this._stopRecording();
         this._stopRecording = undefined;
-        return true;
       }
 
-      return false;
+      return true;
```

Comment:
What's the reason for this change?

Reply:
If `_stopRecording` was falsey, it means recording was never started and is stopped state, so boolean represents if it's currently in a stopped state

Reply:
true! makes sense 👍